test/web3/schema.json

```diff
@@ -0,0 +1,163 @@
+{
+    "params": "0xb3b20624f8f0f86eb50dd04688409e5cea4bd02d700bf6e79e9384d47d6a5a35",
+    "params1": [],
+    "params2": ["0xfe704947a3cd3ca12541458a4321c869"],
+    "params3": [
+      "0xbb3a336e3f823ec18197f1e13ee875700f08f03e2cab75f0d0b118dabb44cba0"
+    ],
+    "params4": ["params", "0x0"],
+    "params5": ["0x29c", "0x0"],
+    "params6": ["params"],
+    "params7": "0x5BAD55",
+    "params8": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675",
+    "params9": "0xc94770007dda54cF92009BFF0dE90c06F603a09f",
+    "submitworkparams": [
```

Comment:
Maybe make these property names camel case or title case, like (for example `"FilterChange"`)